tensorrt7.2/tensorrt/meta.yaml

```diff
@@ -0,0 +1,123 @@
+{% set trt_tar = "TensorRT-7.2.1.6.CentOS-7.6.x86_64-gnu.cuda-11.0.cudnn8.0.tar.gz" %}  #[x86_64]
+
+{% set trt_tar_dir =  environ.get('LOCAL_SRC_DIR', 0) %}
+
+package:
+  name: tensorrt
+  version: {{ tensorrt_version }}
+
+source:
+  - url: file:/{{ trt_tar_dir }}/{{ trt_tar }}
+    sha256: {{ sha256 }}                        
+    patches:
+       # 02xx - GPU only patch specific to open-ce (maybe)
+       - 0204-update-Makefile.conf-for-conda-builds.patch
+       - 0205-update-default-data-path.patch
+
+       # 03xx - patch temporary to fix a problem that when fixed upstream can be removed
+       - 0303-fix-yolov3-for-py3x.patch 
+    folder: tensorrt
+
+  - git_url: "https://github.com/NVIDIA/TensorRT.git"
+    git_tag: 7.2.1
+    patches:
+       # 02xx - GPU only patch specific to open-ce (maybe)
+       - 0201-add-build-onnx2trt-flag.patch
+       - 0202-only-install-onnx2trt.patch
+       - 0203-Fixed-samples-CMakeLists-for-conda-builds.patch
+
+       # 03xx - patch temporary to fix a problem that when fixed upstream can be removed
+       - 0301-fix-GLIBC_2.14-link-issue-with-libnvinfer.so.patch # [x86_64]
+       - 0302-Fix-libcaffeparser-name-for-samples.patch
+       - 0304-modified-print-function-syntax-as-per-py3.patch
+       #- add-makefiles-for-new-samples.patch
+    folder: TensorRT
+
+build:
+  number: 3
+  string: cuda{{ cudatoolkit | replace(".*", "") }}_py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }}
+  activate_in_script: True
+  script_env:
+    - CUDA_HOME
+
+outputs:
+   - name: tensorrt
+     script: build-tensorrt.sh
+     requirements:
+         build:
+           - {{ compiler('cxx') }}
+           - cmake {{ cmake }}
+           - make {{ make }}
+         host:
+           - python {{ python }}
+           - protobuf {{ protobuf }}
+           - libprotobuf {{ protobuf }}
+           - cudnn {{ cudnn }}
+           - libmemcpy_wrapper            #[x86_64]
+           - libclock_gettime_wrapper     #[x86_64]
+
+         run:
+           - python {{ python }}
+           - cudatoolkit {{ cudatoolkit }}
+           - protobuf {{ protobuf }}
+           - libprotobuf {{ protobuf }}
+           - cudnn {{ cudnn }}
+           - numpy {{ numpy }}
+           - uff {{ uff }}
+           - graphsurgeon {{ graphsurgeon }}
+           - onnx-graphsurgeon {{ onnx_graphsurgeon }}
+           - libmemcpy_wrapper            #[x86_64]
+           - libclock_gettime_wrapper     #[x86_64]
+
+   - name: tensorrt-samples
+     script: build-tensorrt-samples.sh
+     requirements:
+         build:
+           - {{ compiler('cxx') }}
+           - cmake {{ cmake }}
+           - make {{ make }}
+         host:
+           - python {{ python }}
+           - protobuf {{ protobuf }}
+           - libprotobuf {{ protobuf }}
+           - cudnn {{ cudnn }}
+           - tensorrt {{ tensorrt_version }}
+           - libmemcpy_wrapper           #[x86_64]
```

Comment:
I don't think the new wrapper libs should be needed, since the tensorrt packages should be pulling them in.

Reply:
yea